index.html

```diff
@@ -4215,53 +4215,77 @@ <h3>Vertical text</h3>
 </section>
 
 
+<section id="typ_bidi_styling" class="subtopic">
+<h3>Bidi styling</h3>
 
+<p class="reviewComments"><a href="https://github.com/w3c/i18n-activity/labels/t%3Abidistyling" target="_blank">See related review comments.</a></p>
 
+<div class="req" id="layout_mirroring">
+	<p class="advisement">Specifications defining user agent-provided user interface controls SHOULD support automatic mirroring for right-to-left languages, ensuring that spatial relationships, navigation patterns, and directional elements are appropriately reversed while preserving semantic meaning.</p>
```

Comment:
This is written in an LTR centric way. Also "automatic mirroring" is overly suggestive? Perhaps:

```suggestion
    <p class="advisement">User interfaces, including user agent provided controls, SHOULD be specified so that the progression of controls is logically ordered and proceeds according to the block progression direction.</p>
    
    <p>For example, a control that displays an icon to the left of some text in a left-to-right direction layout will display the icon to the right of some text in a right-to-left direction layout.</p>
```

Reply:
I think "logically ordered" and "block progression direction" is a bit abstract and could be simplified.

Reply:
```suggestion
    <p class="advisement">User interfaces, including user agent provided controls, SHOULD be specified so that the layout of controls is in [=logical order=] and proceeds according to the [=block direction=].</p>
    
    <p>For example, a control that displays an icon to the left of some text in a left-to-right direction layout will display the icon to the right of some text in a right-to-left direction layout.</p>
```